pages/index.tsx

```diff
@@ -77,7 +77,7 @@ export default function Home() {
               alt="Dashed eye representing data privacy."
             />
             <div className="mt-4 mb-2 text-black text-xl font-bold">End-to-end privacy</div>
-            <div>Trustless to intermediaries as data is encrypted for specific receivers.</div>
+            <div>With end-to-end encryption, there is no need to trust intermediary actors.</div>
```

Comment:
The other way around, we want to emphasis on the result not the how, e.g. :
There is no need to trust intermediaries, as the data is protected from end-to-end...